From a99f40bbda31e0c9a95214f709310c39769ba56d Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Mon, 5 Nov 2018 11:25:57 +0000
Subject: [PATCH 25/26] lineage: Consider techpack/audio when generating kernel
 headers

 * New fancy kernels have techpack modules and we need to
   account for headers of those modules.

Change-Id: I4da9606d5933209287dd5e881f06ee1e39443f70
---
 build/soong/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/soong/Android.bp b/build/soong/Android.bp
index 8210c561..2bb2836f 100644
--- a/build/soong/Android.bp
+++ b/build/soong/Android.bp
@@ -35,7 +35,7 @@ lineage_generator {
 
     // Sources for dependency tracking
     dep_root: "$(TARGET_KERNEL_SOURCE)",
-    dep_files: [ "Makefile", "include/**/*", "arch/$(KERNEL_ARCH)/include/**/*"],
+    dep_files: [ "Makefile", "include/**/*", "arch/$(KERNEL_ARCH)/include/**/*", "techpack/audio/include/**/*"],
 }
 
 cc_library_headers {
-- 
2.17.1

